### PR TITLE
KIN-315 create dashboard

### PR DESCRIPTION
**Pull request by FernandoJoachin**
[KIN-315](https://loopcrack.atlassian.net/browse/KIN-315) Medium [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem

As the company, I want that the main page of the admin panel to be a dashboard so that the users of the admin panel can access quickly to the desired sections.

### Solution

Deploy a panel as the main page of the administration panel. This panel should provide quick access to the desired sections for admin panel users.

### Requirements checklist

- [x] The dashboard must contain cards that when clicked on redirects the user to each one of the admin panel section.
- [x] When clicked on the kinub logo of the sidebar, the user must be redirected to this dashboard

### Testing

- [x] Go to the route: http://kinub.cm/admin
- [x] Test that all the cards work well
- [x] Check that only the super admin sees the user card appear

## Demo
[](https://postimg.cc/rRdnjKjH)

[KIN-315]: https://loopcrack.atlassian.net/browse/KIN-315?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/Views/admin/homeSection/HomeSection.php [hunk 1]
```diff
@@ -16,9 +16,179 @@
...
+                        <h1>Bienvenido</h1>
+
+                        <div class="row">
+                            <div class="col-md-6">
+                                <div class="card card-animate">
+                                    <div class="card-body">
+                                        <a href="/admin/nosotros">
+                                            <div class="d-flex justify-content-between">
+                                                <div>
+                                                    <p class="fw-medium text-muted mb-0">Sección Sobre Nosotros</p>
```
Comment (CheloChidoone) [inline app/Views/admin/homeSection/HomeSection.php:30]: For us all these subtitles inside this paragraph tags are not necessary for the cards, we think that the icon and title should be enough and this subtitles the way they are now could be a bit confusing, if we still want to have them maybe we can rephrase them better

File: app/Views/admin/homeSection/HomeSection.php [hunk 2]
```diff
@@ -16,9 +16,179 @@
...
+                                                    <h2 class="mt-4 ff-secondary fw-semibold"><i data-feather="users" class="icon-dual"></i> <span>Sobre nosotros</span></h2>
```
Comment (CheloChidoone) [inline app/Views/admin/homeSection/HomeSection.php:31]: Here `nosotros` should have the first letter in uppercase like this `Nosotros`

**Review by CheloChidoone — APPROVED** [review]
Good job @FernandoJoachin, it looks very nice, we left a nitpick change for you to do before merging and also another general comment about the subtitles that you added to each card but for that want we want to see your opinion and some other opinions to see what they think of the subtitles there

**Review by CarlosMay7 — APPROVED** [review]

**Review by Mauriciocr207 — APPROVED** [review]

**Review by LUGAMAFE — APPROVED** [review]
Good Job! @FernandoJoachin

**Pull request merged**